.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,73 @@
+<!-- Thank you for contributing a pull request! Here are a few tips to help you:
+
+1. If this is your first contribution, make sure you've read the Contribution Checklist https://developers.mattermost.com/contribute/getting-started/contribution-checklist/
+2. Read our blog post about "Submitting Great PRs" https://developers.mattermost.com/blog/2019-01-24-submitting-great-prs
+3. Take a look at other repository specific documentation at https://developers.mattermost.com/contribute
+-->
+
+#### Summary
+<!--
+A brief description of what this pull request does.
+-->
+
+#### Ticket Link
+<!--
+If this pull request addresses a Help Wanted ticket, please link the relevant GitHub issue, e.g.
+
+  Fixes https://github.com/mattermost/mattermost-server/issues/XXXXX
+
+Otherwise, link the JIRA ticket.
+-->
+
+#### Checklist
+<!--
+Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.
+-->
+- [ ] Added or updated unit tests (required for all new features)
+- [ ] Has UI changes
+- [ ] Includes text changes and localization file updates
```

Comment:
we currently don't have any localization, so this can be removed for the time being